tutorials/asr/Online_ASR_Microphone_Demo.ipynb

```diff
@@ -29,9 +29,6 @@
                 "BRANCH = 'r1.17.0'\n",
                 "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[asr]\n",
                 "\n",
-                "## Install TorchAudio\n",
```

Comment:
Instead of removing it, just comment that it cannot be installed on cuda 12 for the time being

Reply:
this notebook does not need torchaudio